application/common/models/ResetBase.php

```diff
@@ -36,11 +37,13 @@ public static function tableName()
     public function rules()
     {
         return [
-            [['user_id', 'type', 'code', 'expires', 'created'], 'required'],
```

Comment:
That's interesting. Any idea why it no longer considers `type` a required field?  My only guess is that it sees a default value was provided in the column definition (in the migration file).